addon/components/lt-body.js

```diff
@@ -239,6 +292,57 @@ export default Component.extend({
     this.set('useVirtualScrollbar', fixedHeader || fixedFooter);
   },
 
+  didReceiveAttrs() {
```

Comment:
I didn't know that `didUpdateAttrs()` actually can be used to diff the attrs. Nice one, thanks!